cosmos/converter.py

```diff
@@ -6,6 +6,7 @@
 import copy
 import inspect
 from typing import Any, Callable
+import copy
```

Comment:
```suggestion
```

Reply:
Oeps - that slipped through in the solving the merge from console. thank you